src/main/java/ru/skypro/Main.java

```diff
@@ -5,101 +5,92 @@ public static void main(String[] args) {
 
         //number 1
         System.out.println("Задание 1");
-        int cashInMonth = 15000;
-        int capital = 2_459_000;
-        int conservation = 0;
-        int a = 0;
-        while (conservation < capital) {
-            System.out.println("Месяц " + a + " сумма накоплений равна " + conservation + " рублей.");
-            a++;
-            conservation = conservation + cashInMonth;
-            conservation = conservation + conservation / 100;
-        }
+        int[] arr = new int[3];
+        arr[0] = 1;
+        arr[1] = 2;
+        arr[2] = 3;
+
+        double[] arr2 = new double[3];
+        arr2[0] = 1.57;
+        arr2[1] = 7.654;
+        arr2[2] = 9.986;
```

Comment:
этот массив нужно было объявить вот так
```suggestion
        double[] arr2 = {1.57, 7.654, 9.986};
```